lib/sqlitex/server.ex

```diff
@@ -15,6 +15,10 @@ defmodule Sqlitex.Server do
   {:ok, [[a: 2, b: 2]]}
   iex> Sqlitex.Server.query(:example, "SELECT * FROM t ORDER BY a LIMIT 1", into: %{})
   {:ok, [%{a: 1, b: 1}]}
+  iex> Sqlitex.Server.query(:example, "SELECT * FROM t ORDER BY a LIMIT 2", into: :raw_list)
+  {:ok, [[1, 1], [2, 2]], [:a, :b]}
+  iex> Sqlitex.Server.query(:example, "SELECT * FROM t ORDER BY a LIMIT 2", into: :raw_list, types: true)
+  {:ok, [[1, 1], [2, 2]], [:a, :b], [:INTEGER, :INTEGER]}
```

Comment:
I wonder if we should consider returning a map to make it easier for people to pattern-match on the results? I'm imagining a case where someone wants to get column names but has a shared code-path that was already pattern matching on a `{:ok, values, column_names}`.